metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -116,6 +116,7 @@ class MetalsLanguageServer(
   private val openedFiles = new ActiveFiles(time)
   private val messages = new Messages(config.icons)
   private val languageClient = new DelegatingLanguageClient(NoopLanguageClient)
+  private var clientSupportsDidFocus = false
```

Comment:
I just realized, can we use “focusedDocument.isDefined” instead?

Reply:
ahhhh, good call. Yea, that is way simpler and also does the trick. Just pushed this up.